_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<a href="https://www.gnu.org/licenses/lgpl-3.0.en.html">
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+</a>
+
+This blog post outlines MDAnalysis' proposal to change its license
+to the [Lesser GNU Public License][LGPL] (LGPL v3+).
+
+A summary of our [reasons for proposing this license
+change](#Rationale-for-changing-licenses), alongside upcoming actions
+for [community members and library
+contributors](#How-will-the-relicensing-process-work?) are provided.
+
+
+> ⚠️ **Disclaimer**
```

Comment:
Seems a bit over the top, but this entire topic is quite sensitive, the last thing we want is folks thinking this constitutes legal advice.